package.json

```diff
@@ -1,9 +1,41 @@
 {
   "name": "react-devstack",
-  "version": "0.1.0",
+  "version": "0.1.1",
   "description": "An ultimate development stack built on top of React with Redux",
   "author": "Tomas Weiss <tomasw@salsitasoft.com>",
   "license": "MIT",
+  "scripts": {
+    "build": "babel src -d lib",
+    "build:watch": "npm run build -- --watch",
+    "version": "npm run build",
+    "postversion": "git push && git push --tags",
+    "prepublish": "npm run build"
+  },
+  "author": "Tomas Weiss <tomasw@salsitasoft.com>",
+  "bin": {
+    "react-devstack": "./bin/react-devstack.js"
+  },
+  "license": "MIT",
   "dependencies": {
-  }
+    "babel-cli": "^6.24.1",
+    "babel-core": "^6.24.1",
+    "babel-loader": "^7.0.0",
+    "babel-preset-es2015": "^6.24.1",
+    "babel-preset-react": "^6.24.1",
+    "babel-preset-stage-2": "^6.24.1",
+    "cross-spawn": "^5.1.0",
+    "express": "^4.15.2",
+    "react": "^15.5.4",
```

Comment:
`react` and `react-dom` needs to be dependencies because `babel-preset-react` relies on them.